packages/SwingSet/test/vat-admin/terminate/bootstrap-terminate.js

```diff
@@ -116,7 +138,12 @@ export function buildRootObject(vatPowers) {
 
       // We finally hear about doneP resolving, allowing the bootstrap to
       // proceed to the end of the test. We push the 'done' message to testLog
-      await doneP;
+      try {
+        const v = await doneP;
+        testLog(`done result ${v}`);
```

Comment:
let's add an is-this-an-`Error` field to the output, to confirm that the deserialization is taking place correctly and we're getting a real Error object instead of just a string. Maybe:

```js
testLog(`done result ${v} (Error=${v instanceof Error)`);
```

for both `done result` and `done exception`

Reply:
I was presuming that the stringification producing `Error: <msg>` accomplished that, but I'll put in an explicit check in addition.